common/src/main/kotlin/dev/msfjarvis/claw/common/comments/CommentNode.kt

```diff
@@ -59,14 +64,18 @@ internal class CommentNode(
     result = 31 * result + children.hashCode()
     result = 31 * result + isUnread.hashCode()
     result = 31 * result + indentLevel
+    result = 31 * result + isExpanded.hashCode()
     return result
   }
 }
 
+private val _listItems: MutableStateFlow<List<CommentNode>> = MutableStateFlow(emptyList())
+internal val listItems: StateFlow<List<CommentNode>> = _listItems.asStateFlow()
```

Comment:
IIRC having these scoped at the file level effectively makes them static fields, which introduces the problem of having potentially stale values. Is not possible to scope them at the Composable level?

Reply:
How about having a class - something like `CommentsHandler` (open to suggestion for a better name) which will have the `createListNode`, `updateListNode` and also the `StateFlow`. This can now be scoped at the Composable level.

Reply:
That could also work